org-bookmark-heading.el

```diff
@@ -95,11 +100,9 @@ It should take one argument, the path to the file."
   "Return alist for `bookmark-set' for current `org-mode'
 heading.  Set org-id for heading if necessary."
   (let* ((filename (buffer-file-name (org-base-buffer (current-buffer))))
-         (display-filename (funcall org-bookmark-heading-filename-fn filename))
 	 (heading (unless (org-before-first-heading-p)
                     (org-link-display-format (org-get-heading t t))))
-	 (name (concat display-filename (when heading
-                                          (concat ":" heading))))
+         (name (funcall org-bookmark-heading-name-fn filename heading))
```

Comment:
I noticed the indentation appears to change in the diff, which clued me into the fact that the original file had some tabs in it.  I don't know how they got there.  :)  I just pushed a commit that untabifies the whole file.  Would you mind rebasing please?